.github/workflows/checks.yml

```diff
@@ -85,6 +85,21 @@ jobs:
           languages: javascript
       - name: Perform CodeQL analysis
         uses: github/codeql-action/analyze@04df1262e6247151b5ac09cd2c303ac36ad3f62b # v2.2.9
+  devskim:
+    name: DevSkim
+    runs-on: ubuntu-22.04
+    permissions:
+      security-events: write # To upload SARIF results
+    steps:
+      - name: Checkout repository
```

Comment:
TODO: add & configure [`step-security/harden-runner`](https://github.com/step-security/harden-runner)

Reply:
Resolved by 3a2656dc2074d9af2164bdefd6d3afbf1e697ee3 and fd1004d7327bf81f5471ae01772a54e90c613902